### PR TITLE
Deprecate S2373 in favor of S119

### PR DESCRIPTION
**Pull request by cristian-ambrosini-sonarsource**

### CODE REVIEW

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

File: rules/S2373/metadata.json [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
-  "status": "ready",
+  "status": "deprecated",
   "remediation": {
     "func": "Constant\/Issue",
     "constantCost": "5min"
   },
-  "tags": [
-    "convention"
-  ],
+  "tags": [ ],
```
Comment (csaba-sagi-sonarsource) [inline rules/S2373/metadata.json:9]: Why did you remove the tag?

Comment (cristian-ambrosini-sonarsource) [reply rules/S2373/metadata.json:9]: It's part of the [deprecation procedure](https://github.com/SonarSource/rspec#to-deprecate-a-rule):

> In the metadata.json of the rule you want to deprecate:
> 
> - Remove all tags
> - Remove all quality profiles
> - Change the status to deprecated
> - Fill in the replacementRules array with strings in the form "RSPEC-xxxx" with the rules that deprecate this one, if any.

Comment (csaba-sagi-sonarsource) [reply rules/S2373/metadata.json:9]: Hmm interesting, I did not know that.
